manager/integration/tests/test_ha.py

```diff
@@ -1489,3 +1489,18 @@ def test_engine_crash_for_dr_volume():
     16. Check the data md5sum for the DR volume.
     """
     pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_replica_crash_for_rebuild():
+    """
+    [HA] Test replica rebuild should be triggered if any crashes happened.
+
+    1. Create a longhorn volume with replicas.
+    2. Write random data to the volume and get the md5sum.
+    3. Remove file `volume-head-000.img` from one of the replicas.
+    4. Wait replica rebuild to be triggered.
+    5. Check the good replica numbers and one fault replica.
```

Comment:
Verify a new replica is rebuilt, and the old replica containing the crashed file is state ERROR